core/plate.py

```
@@ -92,6 +92,95 @@ def _macros_by_rules(weight_kg: float, kcal: int, goal: Goal) -> Dict[str, int]:
+    changed = False
+
+    if "HIGH_PROTEIN" in diet_flags:
+        target_protein = max(protein, weight_kg * 2.0)
+        if target_protein > protein:
+            protein = target_protein
+            changed = True
+
+    carb_ceiling: Optional[float] = None
+
+    if "LOW_CARB" in diet_flags:
+        # Стремимся к 25% калорий из углеводов, но не ниже 40 г
```

Ruff (0.13.3)

Comment contains ambiguous `г` (CYRILLIC SMALL LETTER GHE). Did you mean `r` (LATIN SMALL LETTER R)?

(RUF003)